recipes-automotive/sdl-generic-hmi/sdl-generic-hmi_git.bb

```diff
@@ -0,0 +1,33 @@
+SUMMARY = "HTML based reference HMI for SmartDeviceLink"
+DESCRIPTION = "SmartDeviceLink (SDL) is a standard set of protocols and \
+messages that connect applications on a smartphone to a vehicle head \
+unit."
+HOMEPAGE = "https://www.smartdevicelink.com"
+SECTION = "app"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=cf455e8d39d2ded1d85b1c5dea6ec3c5"
+
+SRC_URI = "git://github.com/smartdevicelink/generic_hmi.git;branch=develop"
+
+PV = "0.0.0+git${SRCPV}"
+SRCREV = "96c44b8f488977fe1cf29af1a20898187a982b63"
```

Comment:
Got the warning with this commit 
sdl-generic-hmi-0.0.0+gitAUTOINC+96c44b8f48: sdl-generic-hmi: LIC_FILES_CHKSUM points to an invalid file: /home/phongtran/Works/AGL/build/tmp/work/corei7-64-agl-linux/sdl-generic-hmi/0.0.0+gitAUTOINC+96c44b8f48-r0/git/LICENSE [license-checksum]

I checked [96c44b8](https://github.com/smartdevicelink/generic_hmi/tree/96c44b8f488977fe1cf29af1a20898187a982b63). The LICENSE is not available.

Reply:
You are right, we where on an old commit. Updated the PR with a newer commit where the license is available.

Reply:
> I checked 96c44b8. The LICENSE is not available.

I know I checked this an calculated the checksum but you are right - it is not there.  I must have taken the LICENSE from the master branch, where the file does exist!

Surprised it did not fail the build, but if it's only a warning then we might have overlooked it since we were in the early development stages of the SDL integration when this was sent.